README.rst

```diff
@@ -205,6 +205,95 @@ This will override small parts of the django-celery Admin to enable the
 manual launch of PeriodicTask items.
 
 
+Alternative Celery Scheduling (ETA)
+-----------------------------------
+
+Although Celery allows users to schedule the execution of a task by specifying
+an ETA or a countdown, the implementation has at least one main limitation with
+respect to memory consumption: `all workers try to load all tasks with an ETA,
+potentially leading to a large memory consumption
+<https://github.com/celery/celery/issues/2218>`_.
+
+django-celery-fulldbresult proposes an alternative to regular celery ETA with slightly different
+semantics:
+
+1. When a task is sent with an ETA or a countdown, django-celery-fulldbresult
+   intercepts the task and saves it with a status of `SCHEDULED`.
+
+2. A periodic task checks at a configured interval whether the ETA of a task
+   has expired.
+
+3. Once a task is due, a new task with the same parameters but without an ETA
+   is submitted.
+
+4. The task id of the new task is saved in the result of the original scheduled
+   task and the state of the original scheduled task is set to
+   `SCHEDULED_SENT`.
+
+Configuration
+~~~~~~~~~~~~~
+
+Set this variable in your settings.py file:
+
+::
+
+    DJANGO_CELERY_FULLDBRESULT_SCHEDULE_ETA = True
+
+Then create a periodic task in the Django admin or within your code. For
```

Comment:
Maybe I could create a command that creates a periodic task instead. The thing is that I don't want to create a periodic task in somebody else system without their consent. Also, if the setting is False, then no task should be created.